openqa-trigger-bisect-jobs

```diff
@@ -299,10 +300,19 @@ def main(args):
             ]
         )
 
-        out = openqa_clone(
-            params,
-            args.dry_run,
-        )
+        try:
+            out = openqa_clone(
+                params,
+                args.dry_run,
+            )
+        except subprocess.SubprocessError as err:
+            p_stderr = err.stderr if err.stderr else "No error putput"
+            log.error(f"{type(err).__name__}: {p_stderr} ")
+            if 'the repositories for the below updates are unavailable' not in str(err.output):
+                sys.exit(err.returncode)
+            else:
+                log.debug(f"Ignore {type(err).__name__}")
```

Comment:
What do you mean with "ignore" here?

Reply:
Ignore <whatever_raised_Exception>.

Reply:
So you are instructing the reader to ignore the exception? How about "Terminated with ..."

Reply:
Lets follow up on https://github.com/os-autoinst/scripts/pull/376#discussion_r1953508580